docs/content/guides/getting-started/react-redux.md

```diff
@@ -67,7 +72,7 @@ const ExampleComponent = () => {
           <HotTable
             ref={hotTableComponent}
             beforeChange={onBeforeHotChange}
-            settings={hotSettings}
+            {...hotSettings}
```

Comment:
[We were supposed to _not_ group the settings in a settings object.](https://github.com/handsontable/handsontable/blob/feature/react-docs/docs/README-EDITING.md#react-style-guide)